src/components/FeaturedPromo/FeaturedPromo.tsx

```diff
@@ -47,13 +46,12 @@ export const FeaturedPromo = ({
       itemScope
       itemType="https://schema.org/Article"
     >
-      <Link className="cc-featured-promo__figure" to={url}>
+      <Link className="cc-featured-promo__figure" to={href}>
         <figure className="cc-featured-promo__image">
           <ImageElement
             alt={imageAlt}
             height={imageHeight}
             itemProp="image"
-            sizes={imageSizes}
```

Comment:
`sizes` is needed to make srcSet work correctly right?

Reply:
Yep, but I think it should be set within the component (as the component styles define the width of the image...). Does that make sense?

I just need to add a ticket for defining these sizes (which I'll do if you agree with above)

Reply:
From that point of view it makes total sense to define it within the component 👍 . Just need to consider, are there any contexts where they might present differently? (none I can think of)

I was torn about whether or not to have sizes defined in the library or have them passed thru from the consuming app, but makes sense that it's a part of its styling and gets defined here. This is how it's been done for [Card.tsx](https://github.com/wellcometrust/corporate-components/blob/develop/src/components/Card/Card.tsx).

Further thought, would it be useful to perhaps have a config file listing all of the sizes together ?!?!?!?

e.g.

```javascript
// /src/constants/img-sizes.ts

const IMG_SIZES = {
  card: '(min-width: 1494px) 420px, (min-width: 1024px) 27vw, (min-width: 768px) 37vw, 90vw',
  imageCard: '...',
  galleryThumbnail: '...',
  ...
}

export default SIZES
```

Reply:
I don't think there are any contexts where we allow them to present differently (at least not currently).

Not sure about abstracting the config... I'm not sure where it would be used other than within relevant components (ie. Card.tsx only uses `card` key of `IMG_SIZES`, ImageCard.tsx only uses `imageCard` key of `IMG_SIZES` etc.). 

Seems like the abstraction would cause more confusion than declaring the values within each component file?

Reply:
I was spitballing more than anything - sizes can be a pain to wrap your head around at times so this would really be a means of comparison between component types. No point having config for config's sake